packages/es-components/src/components/patterns/statusTracker/StatusTracker.md

```diff
@@ -0,0 +1,22 @@
+The `StatusTracker` component is used to display success or failure along a given set of steps.
+
+```
+const planStates = ['First step', 'Second step', 'Third step', 'Fourth step'];
+
+<StatusTracker statusArray={planStates} step={3} />
+```
+
+The `isErrorState` prop will display the current step with an error status, showing that progress
+can no longer continue.
+
+```
+import Icon from '../../base/icons/Icon';
+
+const planStates = [
+  <span><Icon name="cog" size={22} /> First Step</span>,
+  <span><Icon name="cog" size={22} style={{fontWeight:'normal'}} /> Second Step</span>,
+  <span><Icon name="bell" size={22} /> Third Step</span>
+];
```

Comment:
Does the above mean there's flexibility in what icon is used underneath each step?

Reply:
And, if there are calls-to-action, would they be presented within the step's corresponding `<span>`?

Reply:
Yeah I just threw those icons in there to show you can add pretty much whatever content you want.

Reply:
If you wanted to add a call to action that's where you could put it, yes.